src/components/NoteForm.js

```diff
@@ -71,11 +75,13 @@ function NoteForm({goalId, repoName, note}) {
             Edit Notes
           </Button>
         )}
-        <Button primary onClick={_handleRepoDeletion}>
-          {" "}
-          Delete
+        <Button primary onClick={_handleCancelEditing}>
```

Comment:
Can you create a ternary here to only show the cancel button when editing.

editing ? cancel : delete